third_party/pcsc-lite/naclport/server/src/google_smart_card_pcsc_lite_server/global.h

```diff
@@ -38,6 +43,30 @@ namespace google_smart_card {
 // already been initialized.
 void InitializeAndRunPcscLiteServer();
 
+class PcscLiteServerGlobal final {
+ public:
+  explicit PcscLiteServerGlobal(pp::Instance* pp_instance);
+  ~PcscLiteServerGlobal();
+  void Detach();
+
+  static const PcscLiteServerGlobal* GetInstance();
+
+  void InitializeAndRunPcscLiteServer();
+
+  void PostReaderInitAddMessage(const char* reader_name, int port,
+                                const char* device) const;
+  void PostReaderFinishAddMessage(const char* reader_name, int port,
+                                  const char* device, long return_code) const;
+  void PostReaderRemoveMessage(const char* reader_name, int port) const;
+
+ private:
+  pp::Instance* GetPPInstance() const;
```

Comment:
nit: s/PP/Pp/
(the existing code in this project capitalizes all abbreviations, and also I think I met this recommendation in some of the Chromium style guides.)